subsys/bluetooth/controller/Kconfig

```diff
@@ -146,6 +146,7 @@ config BT_LLL_VENDOR_OPENISA
 	bool "Use OpenISA LLL"
 	depends on BT_LL_SW_SPLIT && SOC_OPENISA_RV32M1_RISCV32
 	select BT_HAS_HCI_VS
+	select BT_CTLR_LE_ENC_SUPPORT if !BT_CTLR_DATA_LENGTH_CLEAR
```

Comment:
I think BT_CTLR_DATA_LENGTH_CLEAR might be an nrf51-only option. So could possebly skip it.
@cvinayak This option should be made nrf specific  I think.

Reply:
The feature says this: "Enable support for Bluetooth v4.2 LE Data Length Update procedure, up to 251 byte cleartext payloads in the Controller. Encrypted connections are not supported."
We want this option too.... to be able to support extended packet lengths for which the user cannot enable encryption (due to performance.